pages/portfolio/[[...address]].tsx

```diff
@@ -165,6 +165,13 @@ const IndexPage: NextPage = () => {
     router.push(router, undefined, { shallow: true })
   }, [tabValue])
 
+  useEffect(() => {
```

Comment:
Makes more sense to change wherever we direct the user to the portfolio to a portfolio which contains the address in the url of the connected wallet.